recipe/build.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/sh
-
-export CXXFLAGS="-I$PREFIX/include -I$PREFIX/include/ncursesw -static-libstdc++"
+# Get an updated config.sub and config.guess
+cp $BUILD_PREFIX/share/gnuconfig/config.* ./pcre2-10.32
 
 mkdir build
 cd build
-cmake .. -DCMAKE_INSTALL_PREFIX="$PREFIX" -DCMAKE_BUILD_TYPE=Release
+cmake ${CMAKE_ARGS} .. \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+    -DGETTEXT_MSGFMT_EXECUTABLE="${BUILD_PREFIX}/bin/msgfmt"
```

Comment:
Add `-DFISH_USE_SYSTEM_PCRE2=ON`? and use pcre2 from conda-forge?

Reply:
I tried getting this to work but I hadn't found the `FISH_USE_SYSTEM_PCRE2` option and got lost in linker errors, I'll give it a try

Reply:
From my local test, it works! Thanks a lot!